media/admin/cs-CZ.plg_vmshipment_zasilkovna.ini

```diff
@@ -16,7 +16,11 @@ PLG_VMSHIPMENT_PACKETERY_CREATED_ON="Vytvořeno dne"
 PLG_VMSHIPMENT_PACKETERY_ORDER_STATUS="Stav"
 PLG_VMSHIPMENT_PACKETERY_ORDER_TOTAL="Celkem"
 PLG_VMSHIPMENT_PACKETERY_SETTINGS="Nastavení"
-PLG_VMSHIPMENT_PACKETERY_API_PASS="Api heslo"
+PLG_VMSHIPMENT_PACKETERY_API_PASS="API heslo"
+PLG_VMSHIPMENT_PACKETERY_FIND_API_PASS_IN_CS="Heslo API naleznete v klientské sekci Zásilkovny v části <a href=\"https://client.packeta.com/cs/support\" target=\"_blank\">Klientská podpora</a>"
```

Comment:
Do překladů nepatří html kód.  Prosím nahradit zástupným symbolem, jako to např. děláme v Shoptetu